ivy_tests/test_ivy/test_functional/test_core/test_creation.py

```diff
@@ -494,14 +494,19 @@ def test_full_like(
     arrays=st.lists(
         hnp.arrays(dtype=dtype_shared, shape=array_shape), min_size=1, max_size=3
     ),
+    indexing=st.sampled_from(["xy", "ij"]),
     dtype=dtype_shared,
 )
 def test_meshgrid(
     arrays,
+    indexing,
     dtype,
     fw,
 ):
 
+    if fw == "torch" and dtype == "uint16" or "uint32":
```

Comment:
we should not have any checks inside the unit test function body, this goes for others as well, but I will just add this comment.